ImportBlocks/BlockImporter.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Akka.Actor;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Plugins
+{
+    public class BlockImporter : UntypedActor
+    {
+        public class StartImport { public IActorRef BlockchainActorRef; public Action OnComplete; }
+        private IActorRef _blockchainActorRef;
+        private const int BlocksPerBatch = 10;
```

Comment:
Does this affect the import speed? Maybe we should turn `BlocksPerBatch` into a configurable parameter.

Reply:
It has very little impact on import speed. The time to receive the next batch is negligible compared to the time of importing a block. We could make it configurable, but it probably isn’t necessary.